### ISSUE TITLE
No such file or directory error when in-repo add-on does not have app directory

### ISSUE THREAD

**Issue by wireframe** [kategengler/ember-cli-code-coverage#161]
Using v1.0.0-beta.2 of ember-cli-code-coverage add-on, I'm receiving this error when running tests on our project with an in-repo add-on:

```
ENOENT: no such file or directory, scandir '/root/app/frontend/lib/ember-proxy-csrf-token/app/'
Error: ENOENT: no such file or directory, scandir '/root/app/frontend/lib/ember-proxy-csrf-token/app/'
    at Error (native)
    at Object.fs.readdirSync (fs.js:953:18)
    at _walkSync (/root/app/frontend/node_modules/walk-sync/index.js:74:18)
    at walkSync (/root/app/frontend/node_modules/walk-sync/index.js:44:10)
    at Class._getIncludesForDir (/root/app/frontend/node_modules/ember-cli-code-coverage/index.js:191:12)
    at _findInRepoAddons.reduce (/root/app/frontend/node_modules/ember-cli-code-coverage/index.js:175:14)
    at Array.reduce (native)
    at Class._getIncludesForInRepoAddonDirectories (/root/app/frontend/node_modules/ember-cli-code-coverage/index.js:169:37)
    at Class._getIncludes (/root/app/frontend/node_modules/ember-cli-code-coverage/index.js:138:12)
    at Class.included (/root/app/frontend/node_modules/ember-cli-code-coverage/index.js:57:30)
    at Class.superWrapper [as included] (/root/app/frontend/node_modules/ember-cli/node_modules/core-object/lib/assign-properties.js:34:20)
    at project.addons.project.addons.filter.addon (/root/app/frontend/node_modules/ember-cli/lib/broccoli/ember-app.js:555:17)
    at Array.filter (native)
    at EmberApp._notifyAddonIncluded (/root/app/frontend/node_modules/ember-cli/lib/broccoli/ember-app.js:552:47)
    at new EmberApp (/root/app/frontend/node_modules/ember-cli/lib/broccoli/ember-app.js:166:10)
    at module.exports (/root/app/frontend/ember-cli-build.js:7:13)
    at Builder.setupBroccoliBuilder (/root/app/frontend/node_modules/ember-cli/lib/models/builder.js:56:19)
    at new Builder (/root/app/frontend/node_modules/ember-cli/lib/models/builder.js:30:10)
    at BuildTask.run (/root/app/frontend/node_modules/ember-cli/lib/tasks/build.js:15:19)
    at Promise.resolve.then (/root/app/frontend/node_modules/ember-cli/lib/models/command.js:243:46)
    at tryCatcher (/root/app/frontend/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:323:19)
    at invokeCallback (/root/app/frontend/node_modules/ember-cli/node_modules/rsvp/dist/rsvp.js:495:31)
```

Our in-repo add-on is very simple that injects an express middleware into the ember server (modeled after the [ember-cli-inject-live-reload addon](https://github.com/ember-cli/ember-cli-inject-live-reload)).  Here's the add-on directory contents:

```
$ tree frontend/lib/ember-proxy-csrf-token/
frontend/lib/ember-proxy-csrf-token/
├── index.js
└── package.json
```

**Comment by adamjmcgrath** [comment]
This has been resolved by https://github.com/kategengler/ember-cli-code-coverage/commit/9c9a3c048d596c5de1e0955c0245eeba858a08d9#diff-168726dbe96b3ce427e7fedce31bb0bcR196 which will be in the next beta release

**Comment by rwwagner90** [comment]
Just published 1.0.0-beta.3, please try it out!

**Comment by wireframe** [comment]
:+1: our build is green with 1.0.0-beta.3!

Thanks to you all for your support!
